src/views/download/download.scss

```diff
@@ -0,0 +1,84 @@
+@import "../../colors";
+@import "../../frameless";
+
+$developer-spot: $ui-yellow;
+
+#view {
+    padding: 0;
+}
+
+.download {
+    .title-banner {
+        &.masthead {
+            background-color: $developer-spot;
+            padding-bottom: 0;
+
+            h1 {
+                margin: 0 0 2rem 0;
+                text-align: left;
+                color: $ui-white;
+            }
+
+            p {
+                margin: 0;
+                width: $cols6;
+                text-align: left;
+                color: $ui-white;
+
+                a {
+                    border-bottom: 1px solid $ui-white;
+                    color: $ui-white;
+                }
+            }
+
+            .band {
+                $band-color: hsla(55, 100, 70, .15);
+                
+                margin-top: 2rem;
+                background-color: $band-color;
+                padding: 1rem 0;
+            }
+
+            .sub-nav {
+                text-align: left;
+                justify-content: flex-start;
+
+                li {
+                    margin: 0 .5rem 0 0;
+                }
+            }
+        }
+    }
+    
+    .download-content {
+        padding-bottom: 2rem;
+    }
+    
+    .installation {
+        display: flex;
+        flex-wrap: wrap;
+        align-items: flex-start;
+        justify-content: space-between;
+    }
+        
+    .installation-column {
+        max-width: $cols4;
+    }
+    
+    .installation-downloads {
+        list-style: none;
+    }
+    
+    .installation-downloads-item {
+        margin: .25rem;
+        padding: 0;
+    }
+    
+    @media only screen and (max-width: $mobile - 1) {
+        .inner {
+            .installation-column {
+                max-width: $cols2;
```

Comment:
this might be a little narrow for mobile:
<img width="399" alt="screen shot 2017-05-23 at 2 08 40 pm" src="https://cloud.githubusercontent.com/assets/1815772/26368992/5cfdb6fa-3fc1-11e7-8a3b-b1747af69e2b.png">

I'd consider setting width to 100% at this stage:
<img width="399" alt="screen shot 2017-05-23 at 2 09 59 pm" src="https://cloud.githubusercontent.com/assets/1815772/26369068/8cdc235c-3fc1-11e7-82d5-84ae195ef461.png">
(please don't mind that it extends off the end – that is due to the issue referenced above with the `p` in the title banner)